sonic_api.yaml

```diff
@@ -2274,6 +2274,35 @@ definitions:
       ip_addr:
         type: string
         description: IP address of directly attached device to the switch
+  BfdEntry:
+    type: object
+    properties:
+      if_name:
```

Comment:
can this be a ref?

Reply:
BfdEntry is added as ref per neighbor. Should it be something else?

Reply:
i mean ref to the interface such as ip interface?

Reply:
'if_name' name should be sufficient to identify the interface right?